boards/st/stm32l562e_dk/Kconfig.stm32l562e_dk

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Yestin Sun
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_STM32L562E_DK
+	select SOC_STM32L562XX
+
+config BOARD_STM32L562E_DK_STM32L562XX_NS
+	bool
+	select SOC_STM32L562XX
```

Comment:
~~Not resolved~~ please amend commits in future rather than adding to the final commit with an incorrect commit